### PR TITLE
Component upgrades and Migration of the HTTP Client

### PR DESCRIPTION
**Pull request by jamezp**
The first commit simply bumps for the version. The next 2 are component upgrades followed by a requirement to use Java 11. The last commit migrates away from the Apache HTTP Client to the Java HTTP Client.

### CODE REVIEW

File: src/main/java/org/jboss/pull/player/GitHubApi.java [hunk 1]
```diff
@@ -211,15 +170,15 @@ private List<ModelNode> filterNonModifiedPullRequests(List<ModelNode> pulls, Ins
...
-        for (HeaderElement el : header.getElements()) {
-            if ("next".equals(el.getParameterByName("rel").getValue())) {
-                String link = el.getName() + "=" + el.getValue();
-                return link.substring(1, link.length() - 1);
+    private String nextLink(HttpResponse<?> response) {
+        final Optional<String> linkHeader = response.headers().firstValue("Link");
+        if (linkHeader.isPresent()) {
+            for (String headerValue : response.headers().allValues("Link")) {
+                System.out.printf("Link Value: %s%n", headerValue);
+                /*if ("next".equals(el.getParameterByName("rel").getValue())) {
```
Comment (luck3y) [inline src/main/java/org/jboss/pull/player/GitHubApi.java]: Is this return not needed? (the one on L180)

Comment (jamezp) [reply src/main/java/org/jboss/pull/player/GitHubApi.java]: Oops, I missed that as I need to parse it. I was just trying to see what the values were. Nice catch!

Comment (jamezp) [reply src/main/java/org/jboss/pull/player/GitHubApi.java]: Should be fixed now.

**Review by luck3y — APPROVED** [review]

**Comment by luck3y** [comment]
Thanks @jamezp!

**Pull request merged**